mrp_repair_partner_language_report/views/report_mrp_repair_order.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+<template id="report_mrp_repair_order_translate">
+    <t t-raw="translate_doc(doc_ids[0], doc_model, 'partner_id.lang', 'mrp_repair.report_mrprepairorder')"/>
```

Comment:
@pedrobaeza Due to the structure of ```report_mrprepairorder``` https://github.com/odoo/odoo/blob/8.0/addons/mrp_repair/views/report_mrprepairorder.xml using ```t-for``` would generate several calls to ```report.html_container```, thus an invalid HTML page.
This is why I added https://github.com/eLBati/manufacture-reporting/blob/mrp_repair_report_partner_lang/mrp_repair_report_customer_lang/README.rst#known-issues--roadmap